archetypes/learn/module/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+layout: module
+
+# The date represents the date the course was created. Posts with future dates are visible
+# in development, but excluded from production builds. Use the time and timezone-offset
+# portions of of this value to schedule posts for publishing later.
+date: {{ .Date }}
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting the module for review.
+draft: true
+
+# The description summarizes the course. It appears on the Learn home and module index pages.
+description: Here is a brief description of what this module's all about.
+
+# The meta_desc property is used for targeting search results or social-media previews.
+meta_desc: Here is a brief description of what this module's all about.
+
+# The order in which the module appears on the home page.
```

Comment:
I assume this doesn't have to be strictly 1,2,3? I could do 10, 20, 30 to give myself some breathing room to shuffle things in between as I build something out?

Reply:
Yep, that's right.